### ISSUE TITLE
Created ticket tweaks

### ISSUE THREAD

**Issue by ste78** [alces-software/alces-flight-center#41]
There are a few changes I think should be made to the tickets that flight center creates. 

1) The 'Alces Flight Center ticket' text in the subject line is nice, but unfortuantely makes tracking them in RT quite hard as the subjects the same on each one, so I think we need to drop that. Instead something along the lines of 'This ticket was generated by Alces Flight Center' being inserted the first line of the actual ticket  body would be better. 

2) Along the same lines and for the same reason, default ticket subjects should be something like..
`<clustername> : <issue category>` eg. `Barkla: Additional Users / Groups` 
However I think when logging tickets the `<issue category>` of the tickets should be editable, so that the user can modify what the ticket subject is. 

3) If the ticket subject is able to be modified we potentially loose some of the category / issue info. Also it'd be handy if the Alces Flight user that logged the ticket is known, so similar to (1) I'd suggest a fixed section at the beginning of each ticket logged with some info:
```
Alces Flight Center Generated Ticket
Cluster : Barkla
User: Steve Norledge
Category: Additional Users / Group

<ticket info as before> 

```

**Comment by bobwhitelock** [comment]
(1) and (3) should be trivial, and I'll get those changes in and released soon.

For (2), am I correct in thinking that we'll want a new input in the Case form for this, which will default to the selected issue and category names but which the user can change if they want?

**Comment by bobwhitelock** [comment]
(3) has already been done - the category and issue are already included, if these are set (which the issue always is); the user/requestor has been added to be included following the suggestion in https://github.com/alces-software/alces-flight-center/issues/37#issue-289943577.

**Comment by ste78** [comment]
(2) yep you are correct, thats what I was imagining also, probably fix the "Barkla:" part of the subject, but allow the <issue category> text to be changed to something else

**Comment by bobwhitelock** [comment]
@ste78 OK sounds reasonable - because of https://github.com/alces-software/alces-flight-center/issues/37#issuecomment-358948462 tickets now also have a fixed suffix, so we can also have a fixed prefix specifying the cluster and have the core of the ticket subject be user-editable.

**Comment by ste78** [comment]
sounds like #37 merges with this ticket in places, lets just have a shot at improving things in this area, get it live and revisit if we still have problems.

**Comment by bobwhitelock** [comment]
> sounds like #37 merges with this ticket in places, lets just have a shot at improving things in this area, get it live and revisit if we still have problems.

Yep, I've made changes to fix the issues mentioned in #37, just need to make the subjects (partially) editable and then will deploy.

**Comment by bobwhitelock** [comment]
This was all done in https://github.com/alces-software/alces-flight-center/pull/44, but GitHub hasn't picked that up.
